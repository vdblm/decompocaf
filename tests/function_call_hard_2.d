double f(double x) {
    return x + 1.0;
}

int power(int i) {
    int x;
    x = -5;
    x = i - 8;
    if (i > 0)
        power(i - 1);
    else {
        Print(f(3.14));
        return 8;
    }
    Print("1: ", i, ", 2: ", x);
    i = 800 + i;
    return i * 2;
}

int main() {
    power(5);
    return 68;
}