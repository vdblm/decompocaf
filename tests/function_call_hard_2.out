4.13999987
1: 1, 2: -7
1: 2, 2: -6
1: 3, 2: -5
1: 4, 2: -4
1: 5, 2: -3
